src/WaveTrack.cpp

```diff
@@ -763,9 +764,19 @@ Track::Holder WaveTrack::Copy(double t0, double t1, bool forClipboard) const
 
          //wxPrintf("copy: clip_t0=%f, clip_t1=%f\n", clip_t0, clip_t1);
 
-         newClip->Offset(-t0);
-         if (newClip->GetPlayStartTime() < 0)
+         // If we are in the case with an audio clip on the right side with blank space to its left,
+         // we need to calculate the offset slightly differently:
+         //  |    [-----CLIP--|--]             [-|----CLIP----]      |
+         // t0                t1       VS.       t0                  t1
+         //  |----|                            |-|
+         //  OFFSET = clip_t0 - t0            OFFSET = 0
+         if (t0 < clip->GetPlayStartTime() && t1 < clip->GetPlayEndTime()) {
+            wxASSERT(clip_t0 - t0 >= 0); //debugging
+            newClip->SetPlayStartTime(clip_t0-t0); //not sure why Offset doesn't work for this; may want to modify
```

Comment:
`Offset` moves clip beginning on `delta` seconds. Not sure which argument You were trying to pass to it but `Offset(-t0)` should work, and as I said it actually yields same result as `SetPlayStartTime(clip_t0 - t0)` in that case. So what was the case where `Offset` didn't work? If it didn't work for wrong arguments then please revert those changes because they do not change anything